modules/nf-core/tsebra/main.nf

```diff
@@ -0,0 +1,61 @@
+process TSEBRA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/tsebra:1.1.2.4--pyhca03a8a_0':
+        'biocontainers/tsebra:1.1.2.4--pyhca03a8a_0' }"
+
+    input:
+    tuple val(meta), path(gtfs)
+    path hints_files
+    path keep_gtfs
+    path config
+
+    output:
+    tuple val(meta), path("*.gtf"), emit: tsebra_gtf
+    tuple val(meta), path("*.tsv"), emit: tsebra_scores
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args                                     ?: ''
+    def prefix      = task.ext.prefix                                   ?: "${meta.id}"
+    def gtf_arg     = '-g ' + gtfs.collect { "$it" }.join(',')
+    def hints_arg   = '-e ' + hints_files.collect { "$it" }.join(',')
+    def keep_arg    = keep_gtfs                                         ? ( '-k ' + keep_gtfs.collect { "$it" }.join(',') ) : ''
+    def config_arg  = config                                            ? "-c $config"                                      : ''
+    def VERSION     = '1.1.2.4'
```

Comment:
Please add the standard disclaimer that the VERSION string needs updating if bumping the version.

Reply:
Can you kindly help with review of the test data?

https://github.com/nf-core/test-datasets/pull/1176